src/commands/cmd_debug.c

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2018-2021 Redis Labs Ltd. and Contributors
+ *
+ * This file is available under the Redis Labs Source Available License Agreement
+ */
+
+#include "../util/arr.h"
+#include "../redismodule.h"
+#include "../graph/graphcontext.h"
+#include "../module_event_handlers.h"
+
+void ModuleEventHandler_AUXBeforeKeyspaceEvent(void);
+void ModuleEventHandler_AUXAfterKeyspaceEvent(void);
+
+// Global array tracking all extant GraphContexts (defined in module.c)
+extern GraphContext **graphs_in_keyspace;
+extern uint aux_field_counter;
+
+static void Debug_AUX(const char *arg) {
```

Comment:
pass original `argv` and `argc` you might want to pass `argv+1, argc-1` as the first value `AUX` was consumed.